WordPress/src/main/java/org/wordpress/android/ui/uploads/PostUploadNotifier.java

```diff
@@ -353,7 +353,7 @@ void updateNotificationSuccessForPost(@NonNull PostModel post, @NonNull SiteMode
                                                                             isFirstTimePublish, true, false);
             PendingIntent pendingIntent = PendingIntent.getService(mContext, 0, publishIntent,
                                                                    PendingIntent.FLAG_CANCEL_CURRENT);
-            notificationBuilder.addAction(R.drawable.ic_posts_grey_24dp, mContext.getString(R.string.button_publish),
+            notificationBuilder.addAction(R.drawable.ic_posts_white_24dp, mContext.getString(R.string.button_publish),
```

Comment:
Having trouble finding where do we set the tint for this one, any quick pointers @theck13 ?

Reply:
Since the `ic_posts_grey_24dp` drawable used before these changes didn't actually correspond to the color used by the notification and all the other vector icons in `PostUploadNotifier` class are white, I don't think a comment is needed.